docs/updates/2019-04.md

```diff
@@ -0,0 +1,197 @@
+# April 2019
+
+Welcome to the April 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Changing engine expiration time
+
+## Enhancements in catwalk
+
+### Demo app
+
+Looking at the visitor statistics we could see that a lot of people opened and left catwalk without opening a Qlik
+application. One possible explanation to this behavior is that these visitors are curious about the tool but lacking
```

Comment:
I had to read the sentence twice. Adding a comma, would help..
"One possible explanation to this behavior is that these visitors are curious about the tool, but lacking"